contrib/endpoints/src/api_manager/proto/server_config.proto

```diff
@@ -196,6 +194,15 @@ message ServiceManagementConfig {
   int32 refresh_interval_ms = 2;
 }
 
+// Maps service configuration files to their corresponding traffic percentage.
+message ServiceConfigTrafficPercentage {
+  // The service configuration file name
+  string service_config = 1;
+  // The traffic percentage which must be greater than 0.0 and the sum must
+  // equal to 100.0.
```

Comment:
Can we ignore this comment?

    // Maps service configuration IDs to their corresponding traffic percentage.
    // Key is the service configuration ID, Value is the traffic percentage
    // which must be greater than 0.0 and the sum must equal to 100.0.

https://github.com/googleapis/googleapis/blob/master/google/api/servicemanagement/v1/resources.proto#L216